packages/editor-ui/src/views/SettingsLogStreamingView.vue

```diff
@@ -14,154 +13,158 @@ import { deepCopy, defaultMessageEventBusDestinationOptions } from 'n8n-workflow
 import EventDestinationCard from '@/components/SettingsLogStreaming/EventDestinationCard.ee.vue';
 import { createEventBus } from 'n8n-design-system/utils';
 import { useDocumentTitle } from '@/composables/useDocumentTitle';
+import { ref, getCurrentInstance } from 'vue';
+import { useI18n } from '@/composables/useI18n';
 import { usePageRedirectionHelper } from '@/composables/usePageRedirectionHelper';
 
-export default defineComponent({
-	name: 'SettingsLogStreamingView',
-	components: {
-		EventDestinationCard,
-	},
-	props: {},
-	data() {
-		return {
-			eventBus: createEventBus(),
-			destinations: Array<MessageEventBusDestinationOptions>,
-			disableLicense: false,
-			allDestinations: [] as MessageEventBusDestinationOptions[],
-			documentTitle: useDocumentTitle(),
-			pageRedirectionHelper: usePageRedirectionHelper(),
-		};
-	},
-	async mounted() {
-		this.documentTitle.set(this.$locale.baseText('settings.log-streaming.heading'));
-		if (!this.isLicensed) return;
+const environment = process.env.NODE_ENV;
 
-		// Prepare credentialsStore so modals can pick up credentials
-		await this.credentialsStore.fetchCredentialTypes(false);
-		await this.credentialsStore.fetchAllCredentials();
-		this.uiStore.nodeViewInitialized = false;
+const settingsStore = useSettingsStore();
+const logStreamingStore = useLogStreamingStore();
+const workflowsStore = useWorkflowsStore();
+const uiStore = useUIStore();
+const credentialsStore = useCredentialsStore();
+const documentTitle = useDocumentTitle();
+const i18n = useI18n();
 
-		// fetch Destination data from the backend
-		await this.getDestinationDataFromBackend();
+const eventBus = createEventBus();
+const disableLicense = ref(false);
+const allDestinations = ref<MessageEventBusDestinationOptions[]>([]);
```

Comment:
nitpick: keep the order suggested in the document

Reply:
I prefer onMounted and beforeMount above the functions. Seems more natural there.